### ISSUE TITLE
About atoms and nodes

### ISSUE THREAD

**Issue by monsteryi0314** [microsoft/Graphormer#143]
What does atom stand for? Isn't it the node? What are the factors that set its size? Very confusing

**Comment by whu-dft** [comment]
> What does atom stand for? Isn't it the node? What are the factors that set its size? Very confusing

Yes, Graphormer uses nodes to represent atoms. A vector with certain size is used to embedding a atom. In my opinion, the dimention of the vector depends on the complexity of the problem.

**Comment by monsteryi0314** [comment]
> > What does atom stand for? Isn't it the node? What are the factors that set its size? Very confusing
> 
> Yes, Graphormer uses nodes to represent atoms. A vector with certain size is used to embedding a atom. In my opinion, the dimention of the vector depends on the complexity of the problem.

What nodes have to do with atoms is particularly puzzling to me, thanks for the answer.

**Comment by whu-dft** [comment]
> 

I think you just need to replace the embedding with your feature, then it should works. And this would requires some studies on the code implementation of Graphormer. (Note graphormer also takes the edges into consideration, see the following suggestion)

In addition, Graphormer is designed to combine Graph representation of molecule and the architecture of transformer. If you only want to apply the model to a non-molecular domain, I think it will be more beneficial to go to transformer directly.

**Comment by whu-dft** [comment]
> 

Sorry, I haven't try any tests yet and I also don't know any benchmarks for non-molecular problems.
